plugin/adsk/plugin/CMakeLists.txt

```diff
@@ -56,6 +57,7 @@ target_include_directories(${TARGET_NAME}
 target_link_libraries(${TARGET_NAME}
     PRIVATE
         $<$<BOOL:${UFE_FOUND}>:${UFE_LIBRARY}>
+        mayaUsdPreviewSurface
```

Comment:
This would mean we have 2 DSO with the same name. The new library, and the old plugin.
Maybe the name should be "basePxrUsdPreviewSurface" in a way similar to the base commands we use for sharing?